src/components/view/DeviceActivityView/DeviceActivityView.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { BlankView, Card, DateRangeCoordinator, DateRangeTitle, Section, SegmentedControl } from "../.."
+import { DailyDataChart, RelativeActivity, RelativeActivityDayCoordinator } from '../../container'
+import { DailyDataType, language, RelativeActivityDataType } from '../../../helpers'
+import standardDailyDataTypes from '../../../helpers/daily-data-types/standard'
+
+export interface DeviceActivityViewProps {
+    presentation?: "Modal" | "Push"
+    previewState?: "default"
+    colorScheme?: "auto" | "light" | "dark"
+}
+
+export default function DeviceActivityView(props: DeviceActivityViewProps) {
+    let [selectedView, setSelectedView] = React.useState<"daily" | "weekly" | "monthly">("daily");
+
+    let dataTypes = standardDailyDataTypes.map((dataType): RelativeActivityDataType => {
+        return {
+            dailyDataType: dataType.type,
+            color: dataType.color
+        }
+    });
+
+    return (
+        <BlankView showBackButton={props.presentation == "Push"}
+            showCloseButton={props.presentation == "Modal"}>
+            <SegmentedControl
```

Comment:
I'm not a fan of this being used to toggle day/week/month. 
I have no good suggestions for what would look better, just the feedback that this didn't feel intuitive to use. 
Some hover text on the segmentcontrol could help. I'm likely misunderstanding why this works like this and am interested in hearing

Reply:
Hmm...it seems like a pretty standard UI paradigm that i've seen in a few apps - e.g. Apple Health at the top (they support longer views so have 6M / Y)
![IMG_3243](https://github.com/user-attachments/assets/59d2cc6f-80f2-4ed1-bc24-b8c352e4e138)
and in general feels like it's a fairly common UI paradigm to have a segmented control at the top for further delineation of types of views

Or is it the look of the segmented control that bugs you?  

Not sure that hover text would make sense, this is designed to be used on mobile

Reply:
Oh uh I just checked the storybook and the labels appear to be missing entirely...I assume that is the issue

Reply:
@reneefromhold check again - I missed saving the strings-en file at the last minute as I was adding translations for these.  It was supposed to appear like this:
<img width="559" alt="Screenshot 2025-01-07 at 4 04 49 PM" src="https://github.com/user-attachments/assets/3448cb30-98b8-424a-9a25-c4683a0ab9cb" />

Reply:
Yes, that was the missing piece. Looks so much better now